packages/wonder-blocks-modal/components/modal-header.test.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import React from "react";
+
+import expectRenderError from "../../../utils/testing/expect-render-error.js";
+import ModalHeader from "./modal-header.js";
+
+describe("ModalHeader", () => {
+    test("using `subtitle` and `breadcrumbs` should throw", () => {
```

Comment:
Can you add test cases for the valid cases?

Reply:
Added a new test on line 9.